documents/policies/external/Competition Requirements.md

```diff
@@ -25,12 +25,13 @@ An official WCA Competition must:
    2. The WCA Delegate for a competition is responsible for:
       1. Submitting the Delegate Report to the WCA Regulations Committee and the WCA Quality Assurance Committee regarding adherence to WCA Regulations and quality standards during the competition, the overall course of the competition, and any incidents that happened. The report must be submitted within one week of the end of the competition.
       2. Submitting the competition results to the WCA Results Team within one week of the end of the competition.
-         1. Results must be double-checked and should be free of errors before submission.
+         1. Results must be double-checked before submission to minimize errors.
+            1. Exception: Results that have not been double-checked (e.g. instances in which scorecards have been lost) may be submitted at WCA Results Team discretion.
```

Comment:
Results can always be submitted; they may not be accepted at the discretion of WRT. Also re-word for consistency across docs:

```suggestion
            1. Exception: Results that have not been double-checked (e.g. instances in which scorecards have been lost) may be accepted at the discretion of the WCA Results Team.
```